conda_env/specs/binstar.py

```diff
@@ -117,4 +127,18 @@ def packagename(self):
 
     def parse(self):
         """Parse environment definition handle"""
-        return self.name.split('/', 1)
+        split_name = self.name.split('/')
+
+        if len(split_name) == 3:
+            #We have a version
+            self.version = split_name[2]
+            return split_name[0:2]
+        elif len(split_name) == 2:
+            #No version should be user/package
+            return split_name
```

Comment:
This looks like a potentially dangerous change in behavior.  You have gone from a limited number of splits to an unlimited one.  Would split('/', 2) work?